### PR TITLE
Adding a log entry to alert that GraphDB might be down or unavailable.

### PR DESCRIPTION
**Pull request by michaelhidalgo**
Adding a log entry to alert that GraphDB might be down or unavailable.

### CODE REVIEW

File: src/controllers/Search-Controller.coffee [hunk 1]
```diff
@@ -78,11 +81,13 @@ class SearchController
                 if (@searchData.results?)
                   @res.send(@renderPage())
                 else
+                  logger?.info {Error:'There are no results that match your search.'}
```
Comment (DinisCruz) [inline src/controllers/Search-Controller.coffee]: who is this message for?

If it is internal then it shouldn't say 'your'

**Comment by DinisCruz** [comment]
can you add some unit tests to check for these scenarios (see the logging tests to see how to capture those messages)

File: src/controllers/Search-Controller.coffee (hunk 1 above)
Comment (michaelhidalgo) [inline src/controllers/Search-Controller.coffee]: It is internal. How  about?

logger?.info {Error:'There are no results that match the search.',queryId: queryId, filters:filters}

File: src/controllers/Search-Controller.coffee [hunk 2]
```diff
@@ -59,6 +59,9 @@ class SearchController
 
         logger?.info {user: @.req.session?.username, action:'show', queryId: queryId, filters:filters}
 
+        if (not queryId?)
```
Comment (DinisCruz) [inline src/controllers/Search-Controller.coffee]: Looking at this better, why are you using the ```queryId`` to identify that the DB is not available?

Comment (michaelhidalgo) [inline src/controllers/Search-Controller.coffee]: I realized that when the GraphDB is down or unavailable and you go to /show/, you do not get results and the queryId becomes undefined.
Is there a better option to check if it is unavailable from the Search controller?

File: src/controllers/Search-Controller.coffee (hunk 1 above)
Comment (michaelhidalgo) [inline src/controllers/Search-Controller.coffee]: @DinisCruz  please chime in on this :)

**Pull request merged**